api/src/main/java/org/openmrs/scheduler/SchedulerService.java

```diff
@@ -103,6 +103,16 @@
 	@Authorized( { "Manage Scheduler" })
 	public TaskDefinition getTask(Integer id);
 	
+	/**
+	 * @since 2.3.0
```

Comment:
Isn't 2.3.0 already released?

Reply:
ya sure it was released,flipped to 2.4.0